src/frontend/src/styles/main.css

```diff
@@ -45,7 +45,70 @@
 
 */
 
-@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap");
+/* cyrillic-ext */
+@font-face {
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 100 900;
+  font-display: swap;
+  src: url(https://fonts.gstatic.com/s/inter/v12/UcC73FwrK3iLTeHuS_fvQtMwCp50KnMa2JL7W0Q5n-wU.woff2) format('woff2');
```

Comment:
fonts must be self hosted

https://wicki.io/posts/2022-02-goodbye-google-fonts-data-privacy-gdpr/

Reply:
Yes. I agree. We are trying to debug weird screenshot differences that should not happend (because the font should stay the same).